examples/ucp_client_server.c

```diff
@@ -448,7 +448,7 @@ static int send_recv_am(ucp_worker_h ucp_worker, ucp_ep_h ep, int is_server,
     if (is_server) {
         am_data_desc.recv_buf = msg;
 
-        while (!am_data_desc.complete) {
+        while (!am_data_desc.complete && !am_data_desc.is_rndv) {
```

Comment:
1. pls add a comment
2. what if is_rndv=1 from the previous iteration? can we exit before descriptor ready?

Reply:
Fixed if is_rndv=1 from prev iteration. 
`am_data_desc.complete` means that the AM callback was called, but rndv is in progress yet.